### PR TITLE
LESSON 23

### PR DESCRIPTION
**Pull request by victorc60**
image
image

### CODE REVIEW

**Comment by IvanCeban** [comment]
"Submit" button inside "Book appointment" looks weird

File: index.html [hunk 1]
```diff
@@ -276,40 +276,44 @@ <h6  style="color:#737373;">CO Founder</h6>
     <section class="contact-us">
         <div class="container__contact-us">
             <div class="contact-us__card">
-                <div class="header__container--contact-us">
+                <div class="header__container--contact-us" style="align-items: start;">
```
Comment (IvanCeban) [inline index.html:279]: no inline styles please

File: index.html [hunk 2]
```diff
@@ -276,40 +276,44 @@ <h6  style="color:#737373;">CO Founder</h6>
...
-                <form class="contact-us__form" action="#">
-                    <input class="input-form fullName" type="text" placeholder="Enter Full Name">
-                    <input class="input-form email"  type="email" placeholder="example@gmail.com">
+                <form class="contact-us__form" action="" method="get" enctype="text/plain" id="form">
+                        <input class="input-form fullName" id="fullname" type="text" placeholder="Enter Full Name" onkeyup="checkInput()" required>
 
-                    <select class="input-form selectPlease" style="max-width: 291px;" name="please" id="pls">
+                        <input class="input-form email"  type="email" placeholder="example@gmail.com" id="email" onkeyup="checkInput()"  required>
+
+                    <select class="input-form selectPlease" style="max-width: 307px; width: 104%;" name="please" id="select1" onclick="checkSelect(selectOne)">
```
Comment (IvanCeban) [inline index.html:288]: width 104% ? Sounds suspicious. 
And again, no inline styles! You have classes for styling

File: index.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    const submitBtn  = document.getElementById('submit-button');
+    const email = document.getElementById('email');
+    const selectOne = document.getElementById('select1');
+    const selectTwo = document.getElementById('select2');
+    const textArea = document.getElementById('message')
+
+
+
+// The functions that give the colors
+    function inputCorrect(inputName){
```
Comment (IvanCeban) [inline index.js:13]: Very good for extracting logic in functions

File: index.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        inputName.style.boxShadow = 'inset 1px 1px 3px 3px rgba(255, 0, 0, 0.8)';
+    }
+
+
+// Main function that checks if the Name and Email inputs 
+// are completed correct
+
+    // submitBtn.addEventListener('submit', checkInput)
+
+    function checkInput(event){
```
Comment (IvanCeban) [inline index.js:28]: On each "checkInput" you're running both `checkName` and `checkEmail` functions. Which is redundant. 
Try to rethink this solution

**Review by IvanCeban — CHANGES_REQUESTED** [review]
Very good strategy of making everything through functions. 
Pay attention on checkInput method

**Comment by victorc60** [comment]
> "Submit" button inside "Book appointment" looks weird

it's button's title attribute

**Pull request merged**